lib/physicians/resolvers.ts

```diff
@@ -0,0 +1,75 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import { DBErrorCode } from '@lib/db/errors'; // Database errors
+import { PhysicianError } from './errors'; // Physician error
+import { Prisma } from '.prisma/client';
+
+/**
+ * Query all physicians of RCD applicants
+ * @returns All physicians of RCD applicants
+ */
+export const physicians: Resolver = async (_parent, _args, { prisma }) => {
+  const physicians = await prisma.physician.findMany();
+  return physicians;
+};
+
+/**
+ * Create a physician
+ * @returns Status of operation (ok, error)
+ */
+export const createPhysician: Resolver = async (_, args, { prisma }) => {
+  const {
+    input: {
+      firstName,
+      lastName,
+      mspNumber,
+      address,
+      city,
+      province,
+      postalCode,
+      phone,
+      status,
+      notes,
+    },
+  } = args;
+
+  let physician;
+  try {
+    physician = await prisma.physician.create({
+      data: {
+        firstName,
+        lastName,
+        mspNumber,
+        address,
+        city,
+        province,
+        postalCode,
+        phone,
+        status,
+        notes,
+      },
+    });
+  } catch (err) {
+    if (err instanceof Prisma.PrismaClientKnownRequestError) {
```

Comment:
I also added this to make sure that the error is a prisma query engine error so that `err.code` wouldn't fail

Reply:
Good catch